src/vm/node_modules/vmload/vmload-datasets.js

```diff
@@ -337,8 +387,17 @@ function zfs(options, cmd, args, lineHandler, callback)
 
     // we don't expect data on stderr, so treat as a warning
     zfs_child.stderr.on('data', function (data) {
```

Comment:
it's possible (not likely) to get incomplete data here, and possibly break on only half a line... it may be worth it to store all of the stderr then post-process it in the exit handler.